src/modules/auth/auth.service.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ConfigService } from '@nestjs/config';
+import { PassportModule } from '@nestjs/passport';
+import { Test, TestingModule } from '@nestjs/testing';
+import Chance from 'chance';
+
+import { AuthService } from './auth.service';
+import { ApiKeyStrategy } from './strategy/api-key.strategy';
+
+describe('AuthService', () => {
+  let authService: AuthService;
+  const chance = new Chance();
+
+  beforeAll(async () => {
+    const app: TestingModule = await Test.createTestingModule({
+      imports: [PassportModule],
+      providers: [AuthService, ApiKeyStrategy, ConfigService],
+    }).compile();
```

Comment:
I think we don't need `PassportModule` or `ApiKeyStrategy` here

Reply:
updated

Reply:
this is updated now